lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -2174,6 +2178,34 @@ class MailboxDashBoardController extends ReloadableController {
     }
   }
 
+  Future<List<EmailAddress>> getContactSuggestion(String query) async {
+    _getAutoCompleteInteractor = getBinding<GetAutoCompleteInteractor>();
+
+    if (_getAutoCompleteInteractor == null || accountId.value == null) {
+      return <EmailAddress>[];
+    }
+
+    final listEmailAddress = await _getAutoCompleteInteractor!
+      .execute(AutoCompletePattern(word: query, accountId: accountId.value!, limit: 2))
+      .then((value) => value.fold(
+        (failure) => <EmailAddress>[],
+        (success) => success is GetAutoCompleteSuccess ? success.listEmailAddress : <EmailAddress>[]
+      ));
+    log('MailboxDashBoardController::getAutoCompleteSuggestion:listEmailAddress: $listEmailAddress');
+    return listEmailAddress;
+  }
+
+  void searchEmailByFromRecipients(BuildContext context, EmailAddress emailAddress) {
```

Comment:
because `from` is the sender not recipient
```suggestion
  void searchEmailByFromFields(BuildContext context, EmailAddress emailAddress) {
```